src/imio/project/pst/migrations/migrate_to_1_3_1.py

```diff
@@ -169,6 +170,20 @@ def run(self):
         # Reorder css and js
         self.runProfileSteps('imio.project.pst', steps=['cssregistry', 'jsregistry'], run_dependencies=False)
 
+        # message
+        if 'new-version' not in self.portal['messages-config']:
```

Comment:
if 'new-version' in self.portal['messages-config']:
            api.content.delete(self.portal['messages-config']['new-version'])
        add...